hooks/post/30-upgrade.rb

```diff
@@ -54,6 +54,11 @@ def remove_docker_v1_content
   Kafo::Helpers.execute('foreman-rake katello:upgrades:3.0:delete_docker_v1_content')
 end
 
+def remove_gutterball
+  return true unless Kafo::Helpers.execute('rpm -q gutterball')
+  Kafo::Helpers.execute("rpm -e gutterball")
```

Comment:
I think we also need:

- tfm-rubygem-hammer_cli_gutterball
- gutterball-certs
- gutterball
- tfm-rubygem-foreman_gutterball

If users are coming from older katello (and Satellite), there's another wrinkle those packages could be ruby193- and not tfm